MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+                        DataObject="{Binding ContactInfo}">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings ShowHelperText="False"/>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.TextInputLayoutSettings = new TextInputLayoutSettings
+        {
+            ShowHelperText = false,
+        };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Validation label
+The validation label used to display the dataform validation messages such as valid or invalid data. Refer [validation](https://help.syncfusion.com/maui/dataform/validation) to learn more about dataform validation.
+
+## Customize Assistive label using text style
+You can customize the font of assistive labels by setting the `TextColor`, `FontFamily`, `FontSize`, and `FontAttributes` properties of the `DataFormTextStyle` property.
+
+#### Hint label style
+You can customize the text of hint label by setting the TextColor, FontFamily, FontSize, and FontAttributes properties of `LabelTextStyle` in DataFormItem.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}"
+                        GenerateDataFormItem="OnGenerateDataFormItem">
+    <dataForm:SfDataForm.TextInputLayoutSettings>
+        <dataForm:TextInputLayoutSettings ShowHelperText="True">
+            <dataForm:DataFormTextItem.LabelTextStyle>
+                <dataForm:DataFormTextStyle TextColor="CadetBlue" FontAttributes="None"/>
+            </dataForm:DataFormTextItem.LabelTextStyle>
+        </dataForm:TextInputLayoutSettings>
+    </dataForm:SfDataForm.TextInputLayoutSettings>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.LabelTextStyle = new DataFormTextStyle { FontAttributes = FontAttributes.Bold, TextColor = Colors.BlueViolet };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Helper label style
+You can customize the text of helper label by setting the TextColor, FontFamily, FontSize, and FontAttributes properties of `HelperTextStyle` in DataFormItem.
```

Comment:
Customize the text of the helper label by setting the TextColor, FontFamily, FontSize, and FontAttributes properties of the `HelperTextStyle` in the DataFormItem.

Reply:
Content changed